### PR TITLE
templates: create issue templates

### PR DESCRIPTION
**Pull request by mhamwala**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
```
Comment (neeraj-laad) [inline .github/ISSUE_TEMPLATE/bug_report.md]: We have expected behaviour but not observed behaviour. Should we add that?

Comment (helenmasters) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I agree with @neeraj-laad we have in our kitura issue template:
### Expected vs. Actual Behaviour
- [ ] Expected to happen...
- [ ] Actually happened...

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (helenmasters) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Are you really interested in which browser people are using, and what version are you expecting in here? aren't you more interested in which stack people are using, or the version of Appsody they used?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (helenmasters) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I would move this below the "Environment Details"

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mhamwala) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I agree, I think that stack and version would be more useful

**Comment by mhamwala** [comment]
Closing as changes addressed here: https://github.com/appsody/docs/pull/60

**Pull request closed without merging**